go.mod

```diff
@@ -0,0 +1,53 @@
+module code-review-bot
```

Comment:
Consider organizing the dependencies alphabetically for improved readability and to easily spot duplicate or redundant packages, especially as your list grows.